src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/OverloadOperatorEqualsOnOverridingValueTypeEqualsTests.cs

```diff
@@ -166,6 +166,20 @@ public override bool Equals(Object obj)
 ");
         }
 
+        [Fact]
+        public void CA2231_CSharp_RefStruct_NoDiagnostic()
+        {
+            VerifyCSharp(@"
+public ref struct S
+{
+    public override bool Equals(object other)
+    {
+        return false;
+    }
+}
+");
```

Comment:
I am even surprised this test passed without passing the `CSharpParseOptions`, I will retry locally.

Reply:
Does it fail locally? I know the new `VerifyCS` helper would not allow the test to pass in that state, but I'm not sure about the old test helpers that this test class is still using.

Reply:
Yeah the test is green locally but I am still gonna add the parse option, it will be less of a problem for later refactoring.